lib/swagger-module.ts

```diff
@@ -78,7 +78,8 @@ export class SwaggerModule {
     app.use(path, swaggerUi.serveFiles(document, options));
 
     httpAdapter.get(path, (req, res) => res.send(swaggerHtml));
-    httpAdapter.get(path + '-json', (req, res) => res.json(document));
+    const swaggerUrl = options?.swaggerUrl ?? (path + '-json');
```

Comment:
I had this same issue earlier today, and I just manually tested this out. Works like charm!!